site/developer/model-documentation/install-and-initialize-validmind.qmd

```diff
@@ -1,17 +1,17 @@
 ---
-title: "Install and initialize the {{< var vm.developer >}}"
+title: "Install and initialize ValidMind"
```

Comment:
This feels like it simultaneously overloads and underspecifies what the name "ValidMind" signifies. How about being very descriptive and simply using the term we already have — "client library"?